.github/workflows/tests-and-lint.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
+        uses: LizardByte/setup-python-action@v2024.919.163656
```

Comment:
Any specific reason on why this action is changing?

Reply:
CI doesn’t pass on `master` because Python 2.7 was removed from actions/setup-python last year: actions/setup-python#672.

If it were my project, I would have removed Python 2.7 support years ago—let me know if that’s the route you would prefer.

Reply:
Ok, I'll read the code on that to see what it does. The support for python 2 is there because on redhat there are still users using it